stdlib/asyncio/subprocess.pyi

```diff
@@ -19,7 +19,7 @@ class SubprocessStreamProtocol(streams.FlowControlMixin, protocols.SubprocessPro
     def __init__(self, limit: int, loop: events.AbstractEventLoop) -> None: ...
     def pipe_data_received(self, fd: int, data: bytes | str) -> None: ...
 
-class Process:
+class Process[CommOut: (PIPE, int, IO[Any], None), CommErr: (PIPE, int, IO[Any], None)]:
```

Comment:
Looks like PIPE is a value of type `int`, not a type alias. I don't think it's a valid constraint for your type variable.

Reply:
I changed the type to a Literal. My assumption then is that mypy etc. can detect if the PIPE itself is used as opposed to another (unknown) int.

Reply:
The problem isn't int vs literal, it's that PIPE Is a value with that type, not a type, so it can't be used in the constraints.